### ISSUE TITLE
[DOC] Create better info on staking pool contract for validator nodes

### ISSUE THREAD

**Issue by janewang** [near/docs#745]
**Is your feature request related to a problem? Please describe.**
Validator nodes can use their on staking contract

**Describe the solution you'd like**
Better documentation on this https://github.com/near/core-contracts/tree/master/staking-pool to foster innovation

**Comment by zavodil** [comment]
Yep, this could be a game changer. Unclear whitelisting process for custom staking pool contract is the blocker here (from my perspective).
